runtime/src/bank/tests.rs

```diff
@@ -7712,6 +7712,10 @@ fn test_bpf_loader_upgradeable_deploy_with_max_len() {
         assert_eq!(*elf.get(i).unwrap(), *byte);
     }
 
+    let loaded_program = bank
+        .load_program(&program_keypair.pubkey())
```

Comment:
This now has to be updated
```suggestion
        .load_program(&program_keypair.pubkey(), false)
```
because I added a parameter to pass debugging_features option to the loader.